docs/source/overview/delivery_services.rst

```diff
@@ -0,0 +1,755 @@
+	| Signed | In all components prior to Traffic Control v2.2. Some endpoints in early versions of the | A boolean value where ``true`` was the same as "url_sig" in current versions, and ``false`` |
+	|        | :ref:`to-api` will still return this field instead of "signingAlgorithm".                | indicated URL signing would not be done for the Delivery Service.                           |
+	+--------+------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------+
+
+Keys for either algorithm can be generated within :ref:`Traffic Portal <tp-services-delivery-service>`.
+
+Static DNS Entries
+------------------
+Static DNS Entries can be added *under* a Delivery Service's domain. These DNS records can be configured in the :ref:`tp-services-delivery-service` section of Traffic Portal, and can be any valid CNAME, A or AAAA DNS record - provided the associated hostname falls within the DNS domain for the Delivery Service. For example, a Delivery Service with xml_id_ "demo1" and belonging to a CDN_ with domain "mycdn.ciab.test" could have Static DNS Entries for hostnames "foo.demo1.mycdn.ciab.test" or "foo.bar.demo1.mycdn.ciab.test" but not "foo.bar.mycdn.ciab.test" or "foo.bar.test".
+
+.. note:: The `Routing Name`_ of a Delivery Service is not part of the :abbr:`SOA (Start of Authority)` record for the Delivery Service's domain, and so there is no need to place Static DNS Entries below a domain containing it.
+
+Tenant
+------
+The :term:`Tenant` who owns this Delivery Service. They (and their parents, if any) are the only ones allowed to make changes to this Delivery Service. Typically, ``tenant``/``Tenant`` refers to the *name* of the owning :term:`Tenant`, but occasionally (most notably in the payloads and/or query parameters of certain :ref:`to-api` requests) it actually refers to the *integral, unique identifier* of said :term:`Tenant`.
+
+.. table:: Aliases
+
+	+----------+----------------------------------------------+--------------------------------------------------------+
+	| Name     | Use(s)                                       | Type(s)                                                |
+	+==========+==============================================+========================================================+
+	| TenantID | Go code and :ref:`to-api` requests/responses | Integral, unique identifier (``bigint``, ``int`` etc.) |
+	+----------+----------------------------------------------+--------------------------------------------------------+
+
+Traffic Router Additional Response Headers
+------------------------------------------
+List of HTTP header ``{{name}}:{{value}}`` pairs separated by ``__RETURN__`` or simply on separate lines. Listed pairs will be included in all HTTP responses from Traffic Router for HTTP-:ref:`Routed <ds-types>` Delivery Services.
```

Comment:
`__RETURN__` is still supported but new lines work as well. can you mention that anywhere where `__RETURN__` is mentioned.

Reply:
Didn't I? I definitely did here, at least - did I miss it somewhere else?